tests/test_jwt_utils.py

```diff
@@ -63,12 +63,6 @@ def test_expired_jwt(self) -> None:
         with pytest.raises(Unauthorized, match="Your token has expired. Please refresh it."):
             decode_jwt(jwt_token)
 
-    @patch("lbz.jwt_utils.get_matching_jwk", MagicMock(return_value={}))
-    @patch("lbz.jwt_utils.ALLOWED_AUDIENCES", [])
-    def test_empty_allowed_audiences(self) -> None:
-        with pytest.raises(RuntimeError, match="ALLOWED_AUDIENCES"):
```

Comment:
In general, the logic remains but we raise a different Exception 🤔 

I would leave these tests and only adjust them, otherwise, we can easily misconfigure lbz by setting a default value for these two variables.